../../pyodbc.pyi,sha256=cLuLanSS0Vry9aUUvn-i1lpDxHiDcMQwQFypGWqD__E,13127
pyodbc-4.0.34.dist-info/INSTALLER,sha256=zuuue4knoyJ-UwPPXg8fezS7VCrXJQrAP7zeNuwvFQg,4
pyodbc-4.0.34.dist-info/LICENSE.txt,sha256=mFnIV3DJG3hmsPdpyTjjbWTQJgFBtSm5R7Tq2WiD6rQ,868
pyodbc-4.0.34.dist-info/METADATA,sha256=zW686PJ2Fw5NeMgwIDdOPPfTyZTJN6YeayV-6mT1XFI,2699
pyodbc-4.0.34.dist-info/RECORD,,
pyodbc-4.0.34.dist-info/REQUESTED,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
pyodbc-4.0.34.dist-info/WHEEL,sha256=TxwUeV-3HEbjjXVQ7qnUZIKQ4IinhZYV_rmkwq84M_Y,102
pyodbc-4.0.34.dist-info/direct_url.json,sha256=whxvOjd_YD6MHqlU4w4FEzL6EdZrf-GF5Pdf8qjt9Q8,70
pyodbc-4.0.34.dist-info/top_level.txt,sha256=30jvYcCul6_wA26OT5c068GAHzqQLpd9WfQ0mSSsDTU,7
pyodbc.cp311-win_amd64.pyd,sha256=kC9xXpKC5qGqBSVaKb5KDWmWL_xPeTeZ2XIsCb1uqFw,131584
